.github/workflows/node.js.yml

```diff
@@ -0,0 +1,39 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install ksc
+        run: |
+          sudo apt-get update
+          curl -fsSL -O https://github.com/kaitai-io/kaitai_struct_compiler/releases/download/0.9/kaitai-struct-compiler_0.9_all.deb
+          sha256sum --check --warn .github/checksums
+          sudo apt-get install ./kaitai-struct-compiler_0.9_all.deb
+          sudo apt-get install --no-install-recommends -y \
+            kaitai-struct-compiler default-jre-headless
+          ksc --version
+      - name: Initialize npm
+        working-directory: javascript
+        run: npm ci
```

Comment:
Hmm, it turns out that `npm ci` works only with `package-lock.json` (https://github.com/kaitai-io/kaitai_compress/runs/3225416347?check_suite_focus=true#step:5:4):

```
Run npm ci

npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.
```

So it needs to be replaced with `npm install`.

Reply:
Done, thanks!